pages/checkout/terms.tsx

```diff
@@ -79,9 +74,9 @@ const CheckoutTermsView: NextPage = () => {
         {/* TODO: Add banner */}
         <BannerContainer>
           <Banner variant="danger">
-            <Title>{intl.formatMessage({ id: "terms.banner.danger.heading1" })}</Title>
+            <Typography variant="h4">{intl.formatMessage({ id: "terms.banner.danger.heading1" })}</Typography>
```

Comment:
Use h2 level here and style the eds h2 according to our design :) And we probably need some space for the p as well.

But as discussed, this one might change if we are only doing a single checkout, so it's not 100% decided what the content in here should be. We will probably at some point need a more generic text content component that can be used in a map function for all the terms for all the data products coming from the API